src/doc/src/reference/unstable.md

```diff
@@ -1259,6 +1259,17 @@ codegen-backend = true
 codegen-backend = "cranelift"
 ```
 
+### `gitoxide`
```

Comment:
Can you update the ToC to include this?

Reply:
A good point. I wasn't sure where to put it so invented the `Other` catch-all category. While at it, I tried to improve the description as well.

All of that is in 5309b61d3ac2672cafedd20a9d06676e1911c07b.